tests/test_telcameras_v2.py

```diff
@@ -128,6 +129,32 @@ class DataPosterTest(APITestCase):
     def setUp(self):
         self.URL = '/telcameras/v2/'
 
+        self.sensor = Sensors.objects.create(objectnummer='GAVM-01-Vondelstraat')
+
+    def test_post_fails_with_non_existing_sensor(self):
```

Comment:
Eerder al aangegeven dat ik niet inzie wat het idee hierachter is, maar los daarvan; de naam van deze test is verwarrend. Hij failed heel expliciet niet.